ui/src/models/workflows.ts

```diff
@@ -211,6 +211,10 @@ export interface Parameter {
      * Description is the parameter description
      */
     description?: string;
+    /**
+     * multi is whether we are having a multi-select choice parameter
```

Comment:
it looks like the comments here might be the same as the API by convention

Reply:
Changed